project/Dependencies.scala

```diff
@@ -37,12 +37,7 @@ object Dependencies {
     crossScalaVersions := Seq(scala212Version, scala213Version),
     scalaVersion := System.getProperty("akka.build.scalaVersion", crossScalaVersions.value.head),
     scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.14.2"),
-    scalaTestVersion := {
-      CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, n)) if n >= 13 => "3.0.8"
-        case _                       => "3.0.7"
-      }
-    },
+    scalaTestVersion := "3.0.8",
```

Comment:
why were we sticking to 3.0.7? DId they make an incompatible change?

Reply:
Hmm, didn't remember there were so many scalatest changes in this PR. Perhaps we should just merge #28473 first, then I'll rebase this one